common/api/summary/appui-abstract.exports.csv

```diff
@@ -3,38 +3,53 @@ Release Tag;API Item
 public;AbstractActionItemProps 
 public;AbstractMenuItemProps 
```

Comment:
Did not deprecate `AbstractMenuItemProps` (together with `AbstractActionItemProps`, `AbstractActionItemProps`, `AbstractMenuItemProps`, `CommonItemProps`) due to `UiAdmin.showContextMenu` and `UiAdmin.showMenuButton`.